### ISSUE TITLE
Create records for regional government councils

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data-constituency-us#2]
According to the [National Association of Regional Councils](http://narc.org/), there are over 100 regional councils, COGs (Council of Governments), and MPOs (Metropolitan Planning Organizations) in the United States. We should import new records for these areas (most, if not all) with polygon geometries and appropriate attributes.

According to the map below, either the [Center for Urban and Regional Policy at Northeastern University](http://www.northeastern.edu/dukakiscenter/) and/or the [National Association of Development Organizations](https://www.nado.org/) have geometry data for these areas. I've reached out and will update the issue when I receive a response.

References:
- Map of council boundaries: https://docs.google.com/viewer?url=http://narc.org/uploads/final_NARC%202.pdf
- List of COGs/MPOs: http://narc.org/about-narc/cogs-mpos/listing-of-cogs-and-mpos/

**Comment by nvkelso** [comment]
Do citizens directly vote for officers of COGS and MPOS? In my experience in California local governments (who's officers _are_ directly voted on) nominate during council sessions representatives to these regional boards. What WOF `placetype` would these things have?

If we're interested in more granular constituencies generally in the same nesting as already imported, US Census "county subdivision" files often have "county" level "supervisor" districts & the like that could be sorted thru and imported.

**Comment by stepps00** [comment]
No, citizens don't typically vote for officers in COGs/councils/MPOs; the goal is to import these with a `"wof:placetype":"constituency"`. 

Right now, we use `"wof:association":"us-senate"` to differentiate between constituency types. "Association" isn't _wrong_, but a different property name (maybe "subtype") would be used to query out the type of constituency; in this case it would be something like `"us-regionalcouncil"`.

**Comment by thisisaaronland** [comment]
cc @migurski

**Comment by migurski** [comment]
The one that I'm initially interested in is SEMCOG, for which we just imported a selection of Michigan addresses to OpenAddresses. We've done a bunch of these as arbitrary geometry outlines in OA, and it'd be nice to have some kind of external ID to refer to similar to the way we use ISO and FIPS codes. Aaron helped me understand a bit about how a COG might be represented.

SEMCOG’s current record: https://github.com/openaddresses/openaddresses/blob/7662413ec/sources/us/mi/semcog.json
